action.yml

```diff
@@ -57,12 +85,12 @@ runs:
         if ($PSNativeCommandArgumentPassing) {
             $PSNativeCommandArgumentPassing = 'Legacy'
         };
-        & "${{ steps.python.outputs.python-path }}" -m pipx run
-        --spec "${{ github.action_path }}"
+        & "${{ steps.cibw.outputs.python-path }}" -m
         cibuildwheel
         "${{ inputs.package-dir }}"
         --output-dir '"${{ inputs.output-dir }}"'
         --config-file '"${{ inputs.config-file }}"'
-        --only '"${{ inputs.only }}"'
+        --only '"${{ inputs.only }}"';
+        exit $LASTEXITCODE
```

Comment:
Why is this needed? Is it currently not exiting correctly?

Reply:
This was needed locally when testing different things with `pwsh` but GHA appends this automatically so  not needed:
https://github.com/actions/runner/blob/1d82031a2cc4bb95ace076743c6cce96ca729881/src/Runner.Worker/Handlers/ScriptHandlerHelpers.cs#L62-L64